staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -127,13 +127,19 @@ func NewReflector(lw ListerWatcher, expectedType interface{}, store Store, resyn
 
 // NewNamedReflector same as NewReflector, but with a specified name for logging
 func NewNamedReflector(name string, lw ListerWatcher, expectedType interface{}, store Store, resyncPeriod time.Duration) *Reflector {
+	realClock := &clock.RealClock{}
```

Comment:
nit - let's not define it here - simply create it in line 142 where you set clock.

Reply:
Both line 140 and 142 needs this clock, so I have to create it ahead.